projects/example-app/src/app/auth/components/logout-confirmation-dialog.component.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatButtonModule, MatDialogModule } from '@angular/material';
+import { LogoutConfirmationDialogComponent } from './logout-confirmation-dialog.component';
+
+describe('Logout Confirmation Dialog', () => {
+  let fixture: ComponentFixture<LogoutConfirmationDialogComponent>;
+  let instance: LogoutConfirmationDialogComponent;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [MatButtonModule, MatDialogModule],
+      declarations: [LogoutConfirmationDialogComponent],
+    });
+
+    fixture = TestBed.createComponent(LogoutConfirmationDialogComponent);
+    instance = fixture.componentInstance;
+  });
+
+  it('should compile', () => {
+    fixture.detectChanges();
+
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should have buttons that are applied [mat-dialog-close]', () => {
```

Comment:
I think this test isn't needed, because we already have a snapshot above.

Reply:
thankyou! I'll remove it.